search_resolution_translation: 0.1 # m 0.04
search_resolution_yaw: 0.3926991 # rad

max_icp_iterations: 10
sensor_resolution_radius: 0.003 #m
min_neighbor_points: 50 #  
